src/main/webapp/dev/js/main/instructorHelp.js

```diff
@@ -1,3 +1,12 @@
+/* global elasticlunr:true */
+
+const questionMap = {};
+const numResultsPerPage = 5;
```

Comment:
Any specific reason why you chose 5 results per page?

Reply:
i was trying to test if the pagination scales well so used small number of results per page. For the final version i was thinking 10 ? Some panels are quite long particularly `Questions` category in which case 10 might be overwhelming ?